base/src/main/java/org/wildfly/security/WildFlyElytronBaseProvider.java

```diff
@@ -26,7 +26,6 @@
 import java.lang.reflect.Constructor;
 import java.security.NoSuchAlgorithmException;
 import java.security.Provider;
-import java.security.Provider.Service;
```

Comment:
Why is this being removed?

Reply:
It wasn't in use, although it's unrelated to my PR, I can revert this change